REPL.cpp

```diff
@@ -140,34 +156,150 @@ bool REPL::ExecuteSwift(std::string line)
     if(ShouldLog(LoggingPriority::Info))
     {
         Log("=========AST Before Modifications==========");
-        src_file->dump();
+        tmp_src_file->dump();
     }
-    AddImportNodes(*src_file, m_modules);
-    //TODO(sasha): Load DLLs
-    swift::performNameBinding(*src_file);
+    AddImportNodes(*tmp_src_file, m_modules);
+
+    swift::performNameBinding(*tmp_src_file);
     CHECK_ERROR();
     swift::TopLevelContext top_level_context;
     swift::OptionSet<swift::TypeCheckingFlags> type_check_opts;
-    swift::performTypeChecking(*src_file, top_level_context, type_check_opts);
+    swift::performTypeChecking(*tmp_src_file, top_level_context, type_check_opts);
     
-    ModifyAST(*src_file);
+    ModifyAST(*tmp_src_file);
     
-    //TODO(sasha): Perform playground transform?
     CHECK_ERROR();
-    swift::typeCheckExternalDefinitions(*src_file);
+    swift::typeCheckExternalDefinitions(*tmp_src_file);
```

Comment:
I would say it would be easier to follow what you were doing here if you had split out the `src_file` to `tmp_src_file` rename into a separate patch.